modal/runner/containers/vllm_unified.py

```diff
@@ -91,39 +105,53 @@ def __init__(
         timeout=10 * 60,
         secrets=[*get_observability_secrets()],
         concurrency_limit=max_containers,
+        keep_warm=keep_warm,
     )
-    return wrap(_VllmContainer)
+    _cls = wrap(_VllmContainer)
+    REGISTERED_CONTAINERS[model_name] = _cls
+    return _cls
+
 
+# A mapping of model names to their respective container classes.
+REGISTERED_CONTAINERS = {}
 
 VllmContainer_MicrosoftPhi2 = _make_container(
     name="VllmContainer_MicrosoftPhi2",
+    model_name="microsoft/phi-2",
```

Comment:
Nice! - I wonder... can we just use the model name for the name of the container itself???

Reply:
i had a similar thought too! just didnt wanna worry about extra logic converting from name -> safe variable name

worth trying out if we keep adding more models frequently